go/memcached-operator/README.md

```diff
@@ -38,6 +38,7 @@ $ go mod tidy
 Build the Memcached operator image and push it to a public registry, such as quay.io:
 
 ```
+$ export GO111MODULE=on
```

Comment:
This is not necessary if using go 1.13, which is new projects should be built with.

Reply:
Thanks for review. and updated.